airgap/terraform/user-data-scripts/provision_registry_server.sh.tpl

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cp ~ubuntu/.ssh/authorized_keys ~root/.ssh/authorized_keys
```

Comment:
Why we need to do this for the root account? Don't we just use the default account to do the ssh operation only in terraform script?

Reply:
The original version of commit had lots of sudo operations in the post-setup scripts, but it's not needed anymore. 
Remove this part and test again.